Ubuntu/installRavKav.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+sudo apt-get install pcscd wine
+sudo cp ravkav "/usr/bin/ravkav"
```

Comment:
This does not look like a good idea. This area of the filesystem should be reserved for system packages.